blocks/form/gdpr.css

```diff
@@ -1,76 +1,34 @@
-.gdpr form input {
-  max-width: 100%;
-  width: 100%;
-}
-
-.gdpr form label {
-  font-size: var(--body-font-size-s);
-}
-
-.gdpr form {
+.gdpr-encrypt form {
   width: 100%;
```

Comment:
I don't think we need this width declaration

Reply:
true, removed it

Reply:
And have you checked if it is needed in other browsers?